internal/client/client.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package client
+
+import (
+	"github.com/go-vela/sdk-go/vela"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/urfave/cli/v2"
+)
+
+// Parse digests the provided urfave/cli context
+// and parses the provided configuration to
+// produce a valid Vela client.
+func Parse(c *cli.Context) (*vela.Client, error) {
```

Comment:
This is where we add the new `Parse()` function.

The purpose of this function is to parse the Vela server address and the token used for authentication. We validate those variables provided and then we create a client setup to communicate with the provided address and token.